registry.json

```diff
@@ -3600,5 +3600,19 @@
     "telegram": "SmaugPool",
     "operatorName": "",
     "github": "SmaugPool"
+  }, 
+  {
+    "ticker": "BOR1",
+    "poolId": "646d4d8214cfd2caa702a072019be4140694aca209ccc92ed8d6bf2a",
+    "social": {
+      "website": "https://ouroboros-pool.com/",
+      "twitter": "PoolOuroboros",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "PoolOuroboros",
```

Comment:
Please can you add a user's TG handle? This must be the handle of a user and not a group or channel.